ui/raidboss/data/05-shb/raid/e8n.js

```diff
@@ -6,8 +6,520 @@
+      response: Responses.getOut(),
+    },
+    {
+      id: 'E8N Scythe Kick',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DE3', capture: false }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DE3', capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DE3', capture: false }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DE3', capture: false }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DE3', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DE3', capture: false }),
+      condition: function(data) {
+        return !data.mirrorsActive;
+      },
+      response: Responses.getUnder(),
+    },
+    {
+      id: 'E8N Biting Frost With Mirror',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DDB', capture: false }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DDB', capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DDB', capture: false }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DDB', capture: false }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DDB', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DDB', capture: false }),
+      condition: function(data) {
+        return data.mirrorsActive;
+      },
+      alertText: {
+        en: 'Get behind, then away from mirrors',
+      },
+    },
+    {
+      id: 'E8N Driving Frost With Mirror',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DDC', capture: false }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DDC', capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DDC', capture: false }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DDC', capture: false }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DDC', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DDC', capture: false }),
+      condition: function(data) {
+        return data.mirrorsActive;
+      },
+      alertText: {
+        en: 'Go Front / Sides, then close between mirrors',
+      },
+    },
+    {
+      id: 'E8N Axe Kick With Mirror',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DE2', capture: false }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DE2', capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DE2', capture: false }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DE2', capture: false }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DE2', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DE2', capture: false }),
+      condition: function(data) {
+        return data.mirrorsActive;
+      },
+      response: Responses.getOutThenIn(),
+    },
+    {
+      id: 'E8N Scythe Kick With Mirror',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DE3', capture: false }),
```

Comment:
Is there a frozen mirror ability id here you could say "get under mirror" when it starts casting? And then this could just say get under? Similar questions about splitting up the mirror casts above as well.

Reply:
There is. For Scythe that wouldn't be a big deal, I suppose, although the opportunity to move after is only about two seconds. 

It's not realistic to do that for Reflected Frost, however, as the user has to anticipate the movement based on the first cast. (The safe zones for either Frost are diametrically opposite, and the user's location at the moment of the initial Frost cast is partly determined by Shiva's facing.)

Reply:
Are the mirrors always in the same spot? (NW/NE) If so, I think you could just have Reflected Frost say "go north" or "go south" then?

Reply:
That is a safe assumption, yes. I'm still not 100% comfortable with not notifying the user prior, but looking at the timeline on FFLogs it does seem to be a 5s cast time between the initial Frost resolution and the Reflected resolution. It definitely feels faster than that, but it could be that's just due to the pressure of eating two vulnerability stacks on failure instead of one.

Reply:
I ran this with triggers in based on reflected.  You're right that on the frosts one, it's a bit tight.  Maybe it's best to leave this as you've written it to give people the best chance of doing the mechanic right.  I think I was just trying to think about how to make the triggers more concise.

Reply:
Thanks for taking the time to verify that. I *really* don't like asking for other people to live test my work for these kinds of things, since that's a far bigger time commitment to ask than a simple code review, but in this case it could just have been that I'm slow on reacting to this specific mechanic. (We all have those 2-3 that cause us problems for no good reason. Curses be laid upon the house of Strike Spark!)